packages/frontend/src/components/composer/AppPickerWrapper.tsx

```diff
@@ -27,11 +27,11 @@ const getJsonFromBase64 = (base64: string): any => {
 
 type Props = {
   onAppSelected?: (app: AppInfo) => void
-  apps: AppInfo[]
-  setApps: (apps: AppInfo[]) => void
 }
 
-export const AppPickerWrapper = ({ onAppSelected, apps, setApps }: Props) => {
+export const AppPickerWrapper = ({ onAppSelected }: Props) => {
+  const [apps, setApps] = useState<AppInfo[]>([])
```

Comment:
Might as well move the state into this component. It's a wrapper after all? The parent component is only interested in what `onAppSelected` calls back with anyway.

Reply:
Initially the idea of the wrapper was, to keep the picker component dumb in the sense it just displays apps. But since we load the icons inside the app picker from the same source, I think now it makes sense to move the fetchApps effect also into the AppPicker and thus get rid of the wrapper component completely